doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -323,6 +323,13 @@ Matter
 
 The issues in this section are related to the :ref:`ug_matter` protocol.
 
+.. rst-class:: v2-5-0 v2-4-0 v2-3-0 v2-2-0 v2-1-0 v2-0-0 v1-9-0 v1-8-0 v1-7-0
```

Comment:
You missed all minor releases here.

Reply:
Also, do not mention versions pro v2.1.1 as Matter was experimental there.